addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -334,7 +334,7 @@ function registerWebsitePreviewTour(name, options, steps) {
         tourSteps.unshift({
             content: "Wait for the edit mode to be started",
             trigger: '.o_website_preview.editor_enable.editor_has_snippets',
-            timeout: 30000,
+            timeout: 60000,
```

Comment:
Is there a test failing a lot because of this?
We have tried to not increase it and instead fix the tests which are approaching the timeout, to prevent a too big timeout that would hide real perf issues (the higher the less we will catch)

Reply:
It was fine on runbot but locally the test would always fail because my machine is slower than runbot.

Reply:
reverted this change